python/ray/tune/tests/test_trial_runner.py

```diff
@@ -413,30 +413,10 @@ def train(config, reporter):
         self.assertEqual(trial.status, Trial.TERMINATED)
         self.assertEqual(trial.last_result[TIMESTEPS_TOTAL], 99)
 
-    def testNoRaiseFlag(self):
-        def train(config, reporter):
```

Comment:
is there some way to keep this?

Reply:
I'm not sure I see what it should be testing since the function runner will always return at least one result, i.e., `{"done": True}`. Let me know what you have in mind and I will be happy to put it back.